onesixtyone.c

```diff
@@ -230,14 +231,14 @@ void read_hosts(char* filename)
         c = 0;
       }
     }
-    else {
-      buf[c++] = ch;
+    else if (ch != '\r') {
+        buf[c++] = ch;
```

Comment:
```suggestion
      buf[c++] = ch;
```
It seems the rest of the code uses 2-spaces indents.